messages/Cargo.toml

```diff
@@ -20,7 +20,7 @@ hex = "0.4"
 num_enum = "0.7.0"
 # We can't pull prio in from the workspace because that would enable default features, and we do not
 # want prio/crypto-dependencies
-prio = { version = "0.14.1", features = ["multithreaded"] }
+prio = { version = "0.15.0", default-features = false, features = ["multithreaded"] }
```

Comment:
This fixes a bug I introduced in #1805: we should not enable default features on `prio` in `janus_messages`.